### PR TITLE
add npm deploy information to travis

### PR DESCRIPTION
**Pull request by GregTheGreek**
I followed [this](https://docs.travis-ci.com/user/deployment/npm/) tutorial to allow travis to auto deploy to npm. 

Should work 🤞

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-  - yarn test
-  - yarn run coverage
+- yarn run lint
+- yarn test
+- yarn run coverage
+deploy:
+  provider: npm
+  email: $GREG_EMAIL 
+  api_key:
+    secure: N+HqEKoJ/9Dc6M6554ZtM8LuPGms++jC0KvykA0GJyVAOmpzdmDlvODdnzfoMQEJfyS0YeL/x9hoWJoEUad2/AlDOyLn+ltd+0y8TLOMgsdwP1KPLGrR07YMgwJZzTDaIEsIAoriuaZp1zXnb7ZFGjSaHgwCUbM5GoRD6+3B42MnrWDxmjvf9aos9ojtN5G5ZTRtnJemcA5Jc8Eeg7KIGgy8nKF6dQHPdTc5oiVp/3UmIMnrZlTrKP4szh6ZZSqj+L818ZSrP4TyhX7OuBz8lLbRImIqOCrBBVA1tGpTcL6zcebxIIE5JnwS4u8RzGDYpqzQqp6P6orllwf+9kLFkYZ8Ou7ezswLuhtTX6XnLjDDlKdJtg7dIyVZGAt1NtnP7VEmJCGh7v1+B24u62128IEYWYyzKeDpY0yWcq+WRmNncDaPPZAwfIGiLclH9923Kb2rhe1GG9hNieWJKJpxLg8vOs/02ouaZhq6rsb5qb11I+q4u7YAEzDBk7qj2N9rEEDdFgqW/PnkltMp5pxoU5OY2NoC79s4U1BdueT2dFrlpfHr6i60Vff2ttq30uZyRi51vTQhEb0/soXvifN8O2sR+qVgrIYn7JYQUTp1Js7KVtl0Dihwa0ohfDxmH2wMpWfThZZwONSwRwsMxvtJxLwf8HYfi3BxWz+91bZ7lgM=
```
Comment (mpetrunic) [inline .travis.yml:19]: this is encrypted key for npm registry, right?

**Review by mpetrunic — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (GregTheGreek) [reply .travis.yml:19]: yea! Its a key generated from my account

**Comment by mpetrunic** [comment]
is this working? I think you are missing `script: yarn build` in deploy stage. we might want to squash or rebase to avoid all those useless commit 😅

**Comment by GregTheGreek** [comment]
@mpetrunic wooops

**Comment by mpetrunic** [comment]
you still didn't put script tag in deploy stage. my reasoning was that by default it executes npm build and therefore its missing dependencies

**Comment by GregTheGreek** [comment]
@mpetrunic that was silly... i completely didn't realize that

**Review by mpetrunic — APPROVED** [review]

**Comment by GregTheGreek** [comment]
@wemeetagain

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**